examples/iris-classification/pinned.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.11.2
```

Comment:
This whole file should be auto generated, can we remove it and add it to the gitignore?